app/queries/facility_analytics_query.rb

```diff
@@ -36,46 +36,6 @@ def registered_patients_by_period
     group_by_user_and_date(@registered_patients_by_period, :registered_patients_by_period)
   end
 
-  def follow_up_patients_by_period
```

Comment:
We're not going to be showing this anymore, we'll need to pick a different thing to show here. [Ref](https://simpledotorg.slack.com/archives/C016EPX1M6E/p1596618747015400).

Reply:
Oh my... goodness... ok bye don't feel like you need to come back. Don't call us, we'll call you...